bin/generate-api

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
```

Comment:
the path `/usr/bin/env` is not available in Windows OS. I'm not sure what the first line does but make sure i works in Windows and MacOS

Reply:
it will be ingored in windows but it is required for the file to be recognized as executable js file. https://stackoverflow.com/questions/33509816/what-exactly-does-usr-bin-env-node-do-at-the-beginning-of-node-files